palantirnet/federated-search-demo
# PL-44 Enable easier development on federated search demo project

## JIRA Ticket(s)

- See ticket [PL-44: Enable easier development on federated search demo project](https://palantir.atlassian.net/browse/PL-44).

## Description

- Updates `composer.json` to pull source for our development modules so that we get a fully functional `.git` 
- Symlinks all the relevant modules into a `/src` directory to ease development.
- reorganizes folder structure so that 4 docroots can coexist
- merges (kinda) #9 
 
## Testing instructions

- `vagrant halt && vagrant up --provision` as new hostnames and dbs have been added
- `vagrant ssh`
- `phing build install-all init`
- inspect the contents of the `src` directory, see that each has a `.git` inside.
- visit http://fs-demo.d8.local and test the search box, observe you can see fully formed results from D8 and D7
- visit http://fs-demo.d7.local and test the search box and see the same results as above.
- visit http://fs-demo.d8-1.local (not hooked up to search yet, but should be running)
- visit http://fs-demo.d8-2.local
- visit http://fs-demo.d8-3.local


## Relevant Screenshots/gifs:

Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.

## Remaining tasks



## Additional Notes:

_Anything more to add?_


---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)
